pkg/scheduler/shared_scheduler_test.go

```diff
@@ -0,0 +1,10 @@
+package scheduler
+
+//TODO implement shared testing utilities
+
+// // TestShardScheduler tests the shard scheduler
+// func TestShardScheduler(t *testing.T) {
+// 	testhelpers.NewConfig()
+// 	StartScheduler(nil, nil, nil)
+
+// }
```

Comment:
Future or current task?

Reply:
Thanks Tom, yes, it's my hope to get a rudimentary test running here for the sake of initial implementation.